config

```diff
@@ -30,8 +30,10 @@ ufs_bin="/usr/bin/unionfs"
 uid=$(id -u)
 gid=$(id -g)
 
-ufs_options="-o cow,allow_other,direct_io,nonempty,auto_cache,sync_read,uid=$uid,gid=$gid"
+# Delete flag used to enable deletions from local media dir by enabling RW. Will not write, but will allow delete.
+delete_flag="N" #Set to Y if you want to allow
```

Comment:
Is it possible we can do it with 0 and 1?
We use 0 and 1 in the config for yes or no.

Reply:
You would think I would do that by default LOL I code SQL databaes day in and out and I would never use Y vs 1.